### PR TITLE
Document new and removed deprecated items in 0.5.0 Changelog, tranche 1

### PR DESCRIPTION
**Pull request by LeeTibbert**
This is the first in what I expect will be a series of PRs to handle deprecated items and document
the action.

These are rough edits. Reviewers, please feel free to change wording or move things to better places.

I will follow your lead on my evolution.   Thank you.

### CODE REVIEW

**Comment by LeeTibbert** [comment]
@ekrich  if you have time and interest, could you take a quick look to see if I inserted
the new text in place that you, as senior author, like? Thank you.

I think documents need a consistency of approach that invites at least loose supervision.
The text is all rough draft, to fulfill promises I have made in various places.

**Comment by LeeTibbert** [comment]
My overall plan, if you concur, is to wait until this PR cycles through review and gets merged.
That will give a good template and I will not be adding unhelpful raw text whilst people are
trying to organize the template.

Then, probably while I am waiting for other PRs to clear CI, I hope to pick off some of the
deprecated items you and I have talked about in an Issue and other places and create
a series of new PRs, each updating this 0.5.0 Changelog.   I am hoping that such
background work will converge to a cleaner code base.

**Review by WojciechMazur — APPROVED** [review]

**Pull request merged**
